packages/pluggableWidgets/datagrid-dropdown-filter-web/src/components/__tests__/FilterComponent.spec.tsx

```diff
@@ -175,4 +180,97 @@ describe("Filter selector", () => {
             });
         });
     });
+
+    describe("focus", () => {
+        beforeEach(() => (document.body.innerHTML = ""));
+
+        it("changes focused element when pressing the input", () => {
+            const component = renderTestingLib(
+                <FilterComponent options={defaultOptions} emptyOptionCaption="Click me" />
+            );
+
+            expect(document.body).toHaveFocus();
+            const input = component.getByPlaceholderText("Click me");
+            expect(input).toBeDefined();
+            fireEvent.click(input);
+
+            jest.advanceTimersByTime(10);
+
+            const items = component.getAllByRole("menuitem");
+            expect(items[0]).toHaveFocus();
```

Comment:
Btw, you don't necessarily need to use the return value of `renderTestingLib` to interact with the components in tests. You can do the following:

```
import { render as renderTestingLib, fireEvent, screen } from "@testing-library/react";

...

  it("changes focused element when pressing the input", () => {
    renderTestingLib(
      <FilterComponent options={defaultOptions} emptyOptionCaption="Click me" />
    );
    
    expect(document.body).toHaveFocus();
    const input = screen.getByPlaceholderText("Click me");
    
    ... // etc.
  }
```

You kind of just interact with the 'screen', this also makes it easier to use something like `beforeEach`

Reply:
Fixed in all tests